src/target/riscv/riscv-013.c

```diff
@@ -4331,8 +4333,17 @@ static int riscv013_halt_go(struct target *target)
 				t->state = TARGET_UNAVAILABLE;
 			}
 		}
+
```

Comment:
Detail: This blank line can be removed.

```suggestion
```

Reply:
Yeah, but I like whitespace in some places to separate different parts of the code.